### ISSUE TITLE
Every time I try opening Mod Assistant, this pops up, and I can't use Mod Assistant.

### ISSUE THREAD

**Issue by StillNotSam** [Assistant/ModAssistant#177]
![image](https://user-images.githubusercontent.com/66231263/83369607-c9c53d00-a371-11ea-985b-ff9660c164c5.png)

**Comment by megalon** [comment]
Looks like an error reading your config file.
Can you post it?
1. Go to `C:\Users\canbo\AppData\Local\ModAssistant\` 
1. You should see at least one folder named `ModAssistant.exe_Url_{some numbers and letters}`. If there's more than one folder, sort by Date Modified and open the newest one
1. In that folder, there should be another folder named `1.1.9.0` with the file `user.config` in it
1. Post the `user.config` file here so that I can take a look at it

I suspect that simply deleting the config file and letting MA regenerate it will fix your problem, but it would be good to see what's wrong with it before you do that.

**Comment by StillNotSam** [comment]
When I open the user.config file, it presents me with this,
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <configSections>
        <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" >
            <section name="ModAssistant.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
        </sectionGroup>
    </configSections>
    <userSettings>

**Comment by PaperT1D** [comment]
I am not OP, but I had the same thing happen to me recently. I'm on 1.1.13. I'm not sure what caused this, but I did have a couple "unexpected shutdowns" on my PC prior to it happening (kittens and a loose power cable don't mix) and ModAssistant may or may not have been open for one of the shutdowns. I'm not sure.

In any case, what I found in the `user` config file was nothing but emptiness after `    <userSettings>` on Line 8. If I remember correctly I think it was actually filled with spaces, rather than ending after Line 8. So all the lines existed, but they were "empty". I deleted everything after `    </configSections>`, hoping that on launch everything would regenerate. I received another error, but I think it said exactly what was missing (or that a specific tag was never closed). Opened the file back up, added `</configuration>` to the end, and launched ModAssistant with no problem. Had to redo some settings, but it worked and everything regenerated in the existing file. I imagine deleting the file would work fine, too, but I kinda like having to figure something out to fix it.
